### ISSUE TITLE
.info/connected always returns true

### ISSUE THREAD

**Issue by mattpcaswell** [EddyVerbruggen/nativescript-plugin-firebase#90]
Used to detect whether or not the app is actually connected to the firebase server, it always returns true when subscribed

`firebase.addValueEventListener(callback, ".info/connected");`

**Comment by EddyVerbruggen** [comment]
Can you point me to the iOS/Android docs for this feature please?

**Comment by mattpcaswell** [comment]
Here it is for ios: https://firebase.google.com/docs/database/ios/offline-capabilities#section-connection-state
And here it is for Android: https://firebase.google.com/docs/database/android/offline-capabilities#section-connection-state

**Comment by EddyVerbruggen** [comment]
Thanks, that was actually a bug on Android. On iOS it worked fine. See #97.

Fix available in 3.5.0.

I verified it works correctly in 3.5.0 this way:

```js
    var path = ".info/connected";
    var onValueEvent = function(result) {
      if (result.error) {
        // ..
      } else {
        console.log("connected: " + result.value);
      }
    };

   firebase.addValueEventListener(onValueEvent, path).then(
        function () {
          console.log("firebase.addValueEventListener added");
        },
        function (error) {
          console.log("firebase.addValueEventListener error: " + error);
        }
    );
```
